### DISCUSSION TITLE
Using a Quasar element I'm unable to find in NiceGUI

### DISCUSSION THREAD

**Discussion by polomint278** [zauberzeug/nicegui#4159] [Ideas / Feature Requests]
I want to use a toggle like found in
https://quasar.dev/vue-components/toggle/

However All I can find in NiceGUI is based of
https://quasar.dev/vue-components/button-toggle

which I find a but too large for the card I'm trying to put it in.

Is there any plans to include
https://quasar.dev/vue-components/toggle/
in Nice GUI?

Or there a way to include it?

Currently im using a simple
```
ui.toggle({False: "Not Monitored", True: "Monitored"}, value=item['monitor_status'],
                              on_change=lambda e, i=i: toggle_monitored(i))
```

**Comment by rodja** [comment]
See https://nicegui.io/documentation/switch
